[korean_9_keys] korean 9 keys updating

Cathaylaboratory opened the pull request:
ready for reviewing.
I deleted those rules for old Hangul. Because I found the 9 keys structure is very hard to handle so much letters

keyman-server commented:
Thank you for your pull request. You'll see a "build failed" message until the Keyman team has reviewed the pull request and manually initiated the build process.

Every change committed to this branch will become part of this pull request. When you have finished submitting files and are ready for the Keyman team to review this pull request, please post a "Ready for review" comment.

LornaSIL commented:
Thanks for reducing the number of rules. When I compile the keyboard there are many, many messages
```
korean_9_keys.kmn:522 - hint KM020B0: The store referenced in index() is longer than the store referenced in any()
```
Basically, starting at line 521, you use a store with 9 characters:
```
store(upper_mid) 'YUIHJLBNM'
```
And you have a store with 14 characters:
```
store(g) '교겨갸고거기규구그가개게걔계'
```
The message is referencing this rule and the following rules:
```
'ㄱ' + any(upper_mid) > index(g, 2)
```
You should not index a rule with a different number of characters in the store. Yes, I suspect the first 9 characters will index properly. You should make the number of character match so you don't get these hundreds of messages. These messages are there in order to help you make your keyboard work better for the users.

LornaSIL reviewed (approved):
Thank you for fixing those! Looks good.

LornaSIL commented:
I just realized your online help file isn't displaying. Can you submit a new PR that renames `korea_9_keys.php` to `korean_9_keys.php` ?
You don't need to do any version bump because that file is not part of the keyboard package. Thanks.

Cathaylaboratory commented:
Sorry about this mistake.
I have replaced the file.

BTW I updated the the korean dubeolsik keyboard at the same time.
Is it OK?

Sent from [Proton Mail](https://proton.me/mail/home) for iOS

On Wed, Jul 16, 2025 at 22:25, Lorna Evans ***@***.***(mailto:On Wed, Jul 16, 2025 at 22:25, Lorna Evans <<a href=)> wrote:

> LornaSIL left a comment [(keymanapp/keyboards#3625)](https://github.com/keymanapp/keyboards/pull/3625#issuecomment-3080955019)
>
> I just realized your online help file isn't displaying. Can you submit a new PR that renames korea_9_keys.php to korean_9_keys.php ?
> You don't need to do any version bump because that file is not part of the keyboard package. Thanks.
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/keymanapp/keyboards/pull/3625#issuecomment-3080955019), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BCAJHRSRSMEYO6KFDQ2YRM33I27M5AVCNFSM6AAAAACBTLQ4XCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTAOBQHE2TKMBRHE).
> You are receiving this because you authored the thread.Message ID: ***@***.***>
